packages/stylelint-plugin-slds/src/rules/lwc-token-to-slds-hook/index.ts

```diff
@@ -66,74 +56,106 @@ function isValueFixed(recommendation, decl, parsedValue) {
   return checkFixed.some(isFixed);
 }
 
-function detectRightSide(decl, basicReportProps, autoFixEnabled) {
-  const parsedValue = valueParser(decl.value);
-  // Usage on right side
-  parsedValue.walk((node) => {
-    if (node.type !== 'word' || !node.value.startsWith('--lwc-')) {
-      return;
-    }
-
-    const oldValue = node.value;
+function isVarFunction(node:valueParser.Node): boolean{
+  return (node.type === "function" && node.value === "var" && node.nodes.length>0);
+}
 
-    if (shouldIgnoreDetection(oldValue)) {
-      // Ignore if entry not found in the list or the token is marked to use further
-      return;
-    }
+function isAlreadyFixed(recommendation:string,functionNode:valueParser.FunctionNode, allNodes:valueParser.Node[]): boolean{
+  const hasFixInFirstNode = allNodes[0].type == "word" && allNodes[0].value === recommendation;
```

Comment:
I am not sure but I wanted to see whether this works for 
- Multi line css value
- In cases where we have left hand side replacements (may be its not applicable in this method)